suite-common/formatters/src/tests/cryptoAmountFormatter.test.ts

```diff
@@ -0,0 +1,42 @@
+import { prepareCryptoAmountFormatter } from '../formatters/prepareCryptoAmountFormatter';
+
+const CryptoAmountFormatter = prepareCryptoAmountFormatter({
+    coins: [],
+    locale: 'en',
+    bitcoinAmountUnit: 0,
+    // @ts-expect-error - no need to test it with Intl for now
+    intl: {},
+});
+
+describe('CryptoAmountFormatter', () => {
```

Comment:
Please add tests also for `sats` outputs

Reply:
Forgot to import :D 
https://github.com/trezor/trezor-suite/pull/7401/commits/c6b3e25196aba1c921c526ecccddf2a7fd180b15

Reply:
https://github.com/trezor/trezor-suite/pull/7401/commits/fe12cdc6e7bc8e79ef44e1d76b04151f04c66547